core/src/datayoga_core/blocks/files/read_csv/block.schema.json

```diff
@@ -28,17 +33,12 @@
     "batch_size": {
       "description": "Number of records to read per batch",
       "type": "number",
-      "default": "1000"
-    },
-    "use_header": {
-      "description": "Determines whether or not to use the first line as the header containing column names. If `false`, field names must be specified",
-      "type": "boolean",
-      "default": true
+      "default": 1000
```

Comment:
also add a min so negative numbers won't be supported.

Reply:
✅